.wordlist.txt

```diff
@@ -225,3 +225,10 @@ xz
 YAML
 ZFS
 zpool
+oob
+AnboxStream
+socat
+unix
+gbinder
+Plaform
+webrtc
```

Comment:
We shouldn't add terms to the wordlist that are not actually words.
"oob" should be "OOB" - that one probably needs to be added.
"AnboxStream" is a component - thus `AnboxStream`.
"socat" is a command - thus `socat`.
The correct spelling for "unix" is Unix.
"gbinder" is a library, thus `gbinder`.
"Plaform" is a typo - should be platform.
And "webrtc" must be spelled "WebRTC", or, if you use it in a format description, as code (`webrtc...`)

Reply:
Thanks, fixed.